CriticalMass/MapViewController.swift

```diff
@@ -83,12 +85,43 @@ class MapViewController: UIViewController {
         gpsDisabledOverlayView.set(title: String.mapLayerInfoTitle, message: String.mapLayerInfo)
         gpsDisabledOverlayView.addButtonTarget(self, action: #selector(didTapGPSDisabledOverlayButton))
         view.addSubview(gpsDisabledOverlayView)
+        gpsDisabledOverlayView.addLayoutsSameSizeAndOrigin(in: view)
+        updateGPSDisabledOverlayVisibility()
+    }
+
+    public func presentMapInfo(with configuration: MapInfoView.Configuration) {
+        dismissMapInfo()
```

Comment:
Maybe use a FeatureFlag here if it is not fully implemented

Reply:
I don't think we need a FeatureFlag here since nobody is calling them method for now (as long as I don't commit my debug calls :D )